solver/src/main.rs

```diff
@@ -120,6 +120,12 @@ struct Arguments {
     )]
     min_order_size_one_inch: U256,
 
+    /// The list of disabled 1Inch protocols. By default, the `PMM1` protocol
+    /// (representing a private market maker) is disabled as it seems to
+    /// produce invalid swaps.
+    #[structopt(long, env, default_value = "PMM1")]
```

Comment:
Does an empty list work here for an env variable?

Reply:
It does not :sob:

Reply:
Yea, this has also been a concern for our unsupported tokens list where we have to include a dummy zero address. What could we include here as null in the event we ever open up "PMM1"? If we do not ever plan to exclude "PMM1" from the configuration, maybe its best left as a constant pushed onto this list at runtime.

Reply:
So I noticed that we have other arguments that use `Vec<...>` that in principle also doesn't support empty lists when using `env`. Would you be OK with merging this as is? In theory, if you run `DISABLED_ONEINCH_PROTOCOLS="" solver` right now it would disable the `""` protocol, so basically a no-op.

I see that we already use a similar work around for other parameters in our staging configuration:
```yaml
  unsupportedTokens: '0x0000000000000000000000000000000000000000' # None, work around not supporting empty list.
```

Reply:
https://github.com/gnosis/gp-v2-services/issues/660